Resources/config/routing_pjax.yml

```diff
@@ -147,7 +147,7 @@ admin_roleList:
         _controller: ezsystems.platformui.controller.role:listRolesAction
 
 admin_roleView:
-    path: /role/view/{roleId}
+    path: /role/view/{roleId}/{tabId}
```

Comment:
@dpobel Thanks, but this is the doc I was reading when doing the first attempt (with optional parameters), so not very helpful. The problem wasn't Symfony, but our code. But I can give it another go.